client/tests/test_client.py

```diff
@@ -85,3 +89,15 @@ def test_delete_clients(self):
         self.assertEqual(response.status_code, 302)
         self.assertEqual(Client.objects.count(), 0)
         self.api_client.logout()
+
+    @ddt.data(
+        (0, 'Can not create -ive or zero number of clients.\n'),
+        (-1, 'Can not create -ive or zero number of clients.\n'),
+        (100, 'Dummy clients created successfully.\n')
+    )
+    @ddt.unpack
```

Comment:
Those are fine here but its a good practice to create commands tests underst management tests module.